airflow/www/static/css/main.css

```diff
@@ -295,6 +295,10 @@ label[for="timezone-other"],
   margin-bottom: 15px;
 }
 
+.dag-import-error {
+  white-space: pre;
+}
+
```

Comment:
I wasn't exactly where in here this should live, so this placement is arbitrary 🤷‍♂️

Reply:
You found the "right" place (or as right a place as exists.)